### ISSUE TITLE
[Feature] : Update the Image Classification Model

### ISSUE THREAD

**Issue by AKACHI-4** [OpenCodeEra/ML-X#3]
## Issue Type
- [ ] Bug 
- [X] Feature

## What ?

I would like to contribute in image classification model to this project. It will help categorize images based on their content, making it easier to search and organize visual data.

- ### Algorithm 

Tell us about the algorithm you will use for model. 

- ### Dataset

Feel Free to discuss the dataset you want to use. 

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

<br>
<br>

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by GitHoobar** [comment]
Hey @AKACHI-4 I have previously worked on Image Classification using Neural Networks.
Can you assign this to me.

**Comment by AKACHI-4** [comment]
Hey @GitHoobar, Yah go ahead... 
happy that you're interested. don't forget to add notes in the notebook so that others can also understand it...

**Comment by GitHoobar** [comment]
Ya Surely.
Can you help me as this will be my very first open source contribution.

**Comment by AKACHI-4** [comment]
> Ya Surely. Can you help me as this will be my very first open source contribution.

Yeah, sure! Feel free to ask your questions.

If you are new to open source and contributions, we encourage you to start by contributing here to get the basics of it: https://github.com/OpenCodeEra/First-Contribution

**Comment by GitHoobar** [comment]
How can I fork the image classification repo, or if I want to create a new notebook then?

**Comment by GitHoobar** [comment]
also can I take any dataset like a set of different images from the web...

**Comment by AKACHI-4** [comment]
> How can I fork the image classification repo, or if I want to create a new notebook then?

Hey, each model is not a separate repo. You have to fork the full ML-X repo, and then you can start working on the Image Classification's notebook.

**Comment by AKACHI-4** [comment]
> also can I take any dataset like a set of different images from the web...

Yeah, you can. You have flexibility in this, but it depends on you. coz then you have to preprocess the data first as per its complexity.
